bazel/foreign_cc/BUILD

```diff
@@ -253,9 +253,10 @@ envoy_cmake(
         # Avoid libidn2
         "USE_LIBIDN2": "off",
         # NGHTTP2.
+        # TODO: Select nghttp2_standard/nghttp2_uhv based on the uhv build setting
```

Comment:
@yanavlasov I would like to change the NGHTTP2_LIBRARY/NGHTTP2_INCLUDE_DIR CMake parameters based on the uhv build setting, but the `select()` can't be used here. Is there another way to change these values based on the configuration?